spec/features/admin/merchants/index_spec.rb

```diff
@@ -0,0 +1,19 @@
+require 'rails_helper'
+
+RSpec.describe "Admin Merchant Index Page", type: :feature do
+  before(:each) do
+    @merchant_1 = Merchant.create!(name: "Steve")
```

Comment:
consider using faker to create the Merchant name,etc. `@merchant_1 = Merchant.create!(:merchant)`